apps/communications/contacts/js/views/form.js

```diff
@@ -822,6 +854,12 @@ contacts.Form = (function() {
       formHeader.addEventListener('action', cancelHandler);
       doMatch(contact, callbacks);
     });
+    var title = formTitle.getAttribute('data-l10n-id');
+    if (title === 'editContact') {
+      Contacts.showStatus({
+        id: 'updateContact'
+      });
+    }
```

Comment:
Anyway this is wrong approach, to detect state of 'model' by checking some of attributes of it's 'view'. To detect current state you easily can use ``mode`` variable, for example.